File: 1. Two Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        n=len(nums)
        res=[]
        for i in range(n):
            complement=target-nums[i]
            if((complement in nums) and (i != nums.index(complement))):
                res.append(i)
                res.append(nums.index(complement))
                break
        return res
            

File: 1. Two Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ans;
        int n=nums.size();
        for(int i=0;i<n;i++){
            for(int j=i+1;j<n;j++){
                if(nums[i] + nums[j] == target)
                {
                    ans.push_back(i);
                    ans.push_back(j);
                    break;

                }
            }
        }
        return ans;
        
    }
};
